s3/oracledb-backups/main.tf

```diff
@@ -4,6 +4,8 @@
 
 locals {
   bucket_name = "${var.tiny_environment_identifier}-oracledb-backups"
+  inventory_bucket_name = "${var.tiny_environment_identifier}-oracledb-backups-inventory-s3bucket"
```

Comment:
if this is name for an s3 bucket does it need "s3bucket" in the name? The name won't be applied to anyother resources?